.github/styles/Vocab/Products/accept.txt

```diff
@@ -253,6 +257,7 @@ subcommands
 subgraph
 Subgraph
 Subnet
+SUBNET
```

Comment:
This should not be included.

Reply:
How should we handle the following line from https://raw.githubusercontent.com/ava-labs/avalanche-docs/master/docs/subnets/subnet-upgrade.md?

```
:::warning NETWORK UPGRADES PERMANENTLY CHANGE THE RULES OF YOUR SUBNET.
```

Reply:
Just make it not all caps?

Reply:
Yeah, `Network Upgrades Permanently Change the Rules of Your Subnet.`